content/en/deployment/overview.md

```diff
@@ -8,3 +8,130 @@ categories: ["Canary Analysis", "Features", "Concepts"]
 tags: ["Deploy Strategy", "Canary", "Blue/Green", "Kubernetes"]
 ---
 
+
+
+## What is a Deployment
+Deployment is the encompassing construct that delivers your code to remote environments. A deployment
+can deliver software to a single environment or multiple environments either in sequence or in parallel
+depending on your [configuration]({{<ref "deployment/create-deploy-config" >}}).
```

Comment:
Thanks @aimeeu !

I agree about the use of the term "encompassing construct". I'd actually love to get your input on how we should be talking about things because "deployment" has become an overloaded term, and I was trying to figure out how to explain that to a reader. 

It feels like we have three different definitions of deployment: 

- deployment - The "construct" I was trying to describe here. In other tools, this is called a "pipeline" but we've chosen not to use that terminology. The "deployment" in the armory doesn't actually perform any actions, it's a container that is made up of environment deployments. 
- environment deployment (my term, as I don't know that we actually have an agreed-upon term to describe this) - This is the thing that actually performs actions. At other companies, this is where I'd say something like, "I'm doing a production deployment" It is the set of actions taken to deploy code to a single remote environment. 
- Deployment - the k8s construct. Not really worried about this one, and I understand why we should be specific about "deployment" vs "Deployment". 

Basically, I was trying to explain the concept behind what Armory calls a "deployment" ( a set of 1 or more "environment deployments" configured in sequence or in parallel) vs. what I think most people think of when they read deployment (shipping code to an environment). 

I could absolutely be overthinking this but figured it was worth asking for your thoughts.